### PR TITLE
Update build_base_images to only build one image, tag push others,

### PR DESCRIPTION
**Pull request by ericvn**
The prior PR iterated over the comma separated hubs, and built and pushed each set of images. This could result in different images across the hubs.

This PR will build and push the first HUBs images, and for successive Hubs, retag the images using the new HUB and then push. This will result in matching images across the HUBs.

### CODE REVIEW

**Review by howardjohn — COMMENTED** [review]
You can also set multiple values here: https://github.com/istio/istio/blob/be93c41fa307631c9218e7e85f96ef1dd344cc74/tools/buildx-gen.sh#L94 and buildx will take care of it

**Comment by ericvn** [comment]
@howardjohn I can try making the change there. Plan would be to introduce a new HUBS env var (defaults to HUB) so things continue to work as they are now, and use that in the script.

**Comment by ericvn** [comment]
Updated to use the buildx-gen.sh script and changing build-base-images to only use the new HUBS env var.

I am still missing a piece in `Istio-docker.mk` to update the other targets to also accept HUBS. I have most of that working, but not committed, as `docker.push` is failing (this seemed to be a hard area to make work correctly earlier in the file history).

File: tools/buildx-gen.sh [hunk 1]
```diff
@@ -87,11 +87,19 @@ for file in "$@"; do
       VM_IMAGE_VERSION="${split[-1]}"
     fi
 
+    # Create a list of tags by iterating over HUBs
```
Comment (howardjohn) [inline tools/buildx-gen.sh:90]: I hate bash :slightly_smiling_face:

**Review by howardjohn — APPROVED** [review]

**Comment by ericvn** [comment]
Also, as part of the updating for the non `x` targets, builds are done using separate `docker builds` so needs additional logic to do tags instead of builds for secondary HUBS (else the images may be different) is needed.

**Pull request merged**